src/commands/account/add.ts

```diff
@@ -46,15 +46,11 @@ export default class AccountAdd extends Command {
     };
 
     this.axiosConfig.baseURL = REGIONS_API_URL[region];
```

Comment:
Do we still need this? Let's just remove everything related to Axios.

Reply:
> Do we still need this? Let's just remove everything related to Axios.

As we changed all `axios` requests to `got`, there's no reason to keep axios configuration...
I'll get rid of it.